core/domain/exp_services_test.py

```diff
@@ -1874,6 +1874,11 @@ def test_export_to_zip_file_with_assets(self):
             fs_domain.GcsFileSystem(
                 feconf.ENTITY_TYPE_EXPLORATION, self.EXP_0_ID))
         fs.commit('abc.png', raw_image)
+        with python_utils.open_file(
```

Comment:
Add a comment above this line to say something like "Note that audio files should not be included in asset downloads."